feature/about/src/main/java/org/mifos/mobile/feature/about/ui/AboutUsScreen.kt

```diff
@@ -38,6 +40,7 @@ internal fun AboutUsScreen(
     LazyColumn(
         modifier = modifier
             .fillMaxSize()
+            .background(MaterialTheme.colorScheme.background)
```

Comment:
Don't apply color directly, instead configure theme or core component and it will be reflected on all screens